### PR TITLE
OZ-592: Updated Odoo-OpenMRS flows functional documentation.

### PR DESCRIPTION
**Pull request by VaishSiddharth**
Update details of Odoo-OpenMRS flow and updated the mappings.

JIRA Ticket: https://mekomsolutions.atlassian.net/browse/OZ-592

### CODE REVIEW

File: docs/users/odoo-openmrs.md [hunk 1]
```diff
@@ -48,20 +50,35 @@ flowchart LR
...
     a["OpenMRS billable items"]-- many-to-1 -->b["Odoo quotation"]
 ```
 
-### **3** &nbsp; OpenMRS Billable Item → Odoo Quotation Item
+### **4** &nbsp; OpenMRS Billable Item → Odoo Quotation Item
+
+Each billable item ordered in OpenMRS during a patient's visit is synchronized in Odoo as a quotation item in the corresponding customer's quotation that is linked with the OpenMRS patient's visit.
 
-Each billable item ordered for an OpenMRS patient is synchronized in Odoo as an order line in the corresponding customer's quotation.
+When a drug order is discontinued for a patient in OpenMRS, the corresponding quotation item is removed from the corresponding Odoo quotation.
```
Comment (mks-d) [inline docs/users/odoo-openmrs.md:77]: What happens when a lab order is discontinued? This part wasn't documented in ERPNext-OpenMRS, @corneliouzbett surely the same thing happens?

**Review by mks-d — COMMENTED** [review]
I have refactored it a bit + commented.

Note that the discontinuation of drug orders is not a new flow, it falls into the _OpenMRS Billable Item → Odoo Quotation_ Item flow.

File: docs/users/odoo-openmrs.md (hunk 1 above)
Comment (VaishSiddharth) [reply docs/users/odoo-openmrs.md:77]: Not sure if we can discontinue a lab order. Also as per my understanding a lab order is a one time thing unlike drug order which comes under `Active Medications`
CMIIW

Comment (corneliouzbett) [reply docs/users/odoo-openmrs.md:77]: > @corneliouzbett surely the same thing happens?

Yes, same thing happens.

> What happens when a lab order is discontinued?

The corresponding quotation item is removed from the quotation, and if it was the only item in the quotation, then quotation is marked as cancelled or deleted.

Comment (mks-d) [reply docs/users/odoo-openmrs.md:77]: @VaishSiddharth then, if you don't mind, and if this piece of documentation is missing in the Odoo-OpenMRS flows, could you add it there too as part of this PR?

**Comment by VaishSiddharth** [comment]
Updated the docs please have a look.

File: docs/users/odoo-openmrs.md [hunk 2]
```diff
@@ -9,19 +9,21 @@
         participant Odoo
         OpenMRS->>Ozone: Patients
         Ozone->>Odoo: Customers
+        OpenMRS->>Ozone: Visits
         OpenMRS->>Ozone: Drug orders
         OpenMRS->>Ozone: Lab test orders
         Ozone->>Odoo: Quotations
-        Ozone->>Odoo: Order lines
+        Ozone->>Odoo: Quotation items
```
Comment (corneliouzbett) [inline docs/users/odoo-openmrs.md:16]: I think we should maintain the term 'Order Lines' as it is the terminology used within Odoo. 'Quotation Items' is primarily used in the context of ERPNext.

Comment (VaishSiddharth) [reply docs/users/odoo-openmrs.md:16]: +1
Lets wait for @mks-d to respond on this. I think he renamed `Sale Order Lines` to `Quotation` so maybe he has a preference here.

File: docs/users/erpnext-openmrs.md [hunk 3]
```diff
@@ -77,6 +77,10 @@ flowchart LR
 
 Each billable item ordered in OpenMRS during a patient's visit is synchronized in ERPNext as a quotation item in the corresponding customer's draft quotation that is linked with the OpenMRS patient's visit.
 
+When a drug / lab order is discontinued for a patient in OpenMRS, the corresponding quotation item is removed from the corresponding ERPNext quotation.
+
+If all quotation items are removed from a quotation, the quotation is then marked as cancelled or deleted.
```
Comment (mks-d) [inline docs/users/erpnext-openmrs.md:82]: corneliouzbett can you clarify if the quotation is cancelled or deleted?

File: docs/users/odoo-openmrs.md (hunk 2 above)
Comment (mks-d) [reply docs/users/odoo-openmrs.md]: Done, I have reverted it on @VaishSiddharth's PR branch.

File: docs/users/erpnext-openmrs.md (hunk 3 above)
Comment (corneliouzbett) [reply docs/users/erpnext-openmrs.md:82]: For ERPNext, the quotation is deleted. In Odoo quotation is cancelled.

File: docs/users/odoo-openmrs.md [hunk 4]
```diff
@@ -48,20 +50,35 @@ flowchart LR
 
     <small>_option 1_</small> &nbsp; An OpenMRS patient is always synchronised as an Odoo customer.
```
[the remark this answers is not in the document]
Comment (VaishSiddharth) [reply docs/users/odoo-openmrs.md:51]: Yes

**Review by mks-d — APPROVED** [review]

**Pull request merged**